tox.ini

```diff
@@ -2,123 +2,81 @@
 mintoxversion = 2.3
 envlist = check_readme,flake8,covclean,py38,coverage,integrationtests
 
+[flake8]
+max-line-length = 99
+exclude = .tox,wiki,.cache,.d3a,.hypothesis,.pytest_cache,vagrant,requirements,venv
+
+# Settings defined in the top-level testenv section are automatically inherited by individual
+# environments unless overridden
 [testenv]
-passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH SOLC_BINARY INTEGRATION_TESTS_REPO
+basepython = python3.8
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH SOLC_BINARY INTEGRATION_TESTS_REPO INTEGRATION_TESTS_BRANCH
+allowlist_externals =
+    git
+    /bin/rm
+    /bin/ln
+    bash
 deps =
     pip-tools
```

Comment:
You've removed the usage of `pip sync`, so I think that `pip-tool` is not required as a dependency anymore here. Could you please check?